### ISSUE TITLE
Help command could pin to a specific Ubuntu version?

### ISSUE THREAD

**Issue by alexellis** [weaveworks/ignite#116]
I guess that `weaveworks/ignite-ubuntu` corresponds to `weaveworks/ignite-ubuntu:latest`, but it'd be better I think for DX if it was: "weaveworks/ignite-ubuntu-bionic" or "weaveworks/ignite-ubuntu:bionic" or similar.

```
	$ ignite run weaveworks/ignite-ubuntu \
		--cpus 2 \
		--memory 1024 \
		--ssh \
		--name my-vm
	$ ignite images
	$ ignite kernels
	$ ignite ps
	$ ignite logs my-vm
	$ ignite ssh my-vm
```

**Comment by luxas** [comment]
Good suggestion. Let's fix it!

**Comment by luxas** [comment]
We probably won't have time to fix this in v0.4.1, will move to v0.4.1. It's not release blocking (we're doing a release with bugfixes today)
